NameLayerMain/src/vg/civcraft/mc/namelayer/command/commands/TransferGroup.java

```diff
@@ -52,12 +52,6 @@ public boolean execute(CommandSender sender, String[] args) {
 			return true;
 		}
 		
-		if (!gm.hasAccess(g, uuid, PermissionType.getPermission("TRANSFER")) && 
```

Comment:
Uh wait yeah, you are right, I'll add that. Do you agree that permanently binding it to primary owner is better than the permission?